crossfit/utils/model_adapter.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, Callable
+
+
+def adapt_model_input(model: Callable, encoded_input: dict) -> Any:
```

Comment:
Function looks good 👍

Reply:
Thanks a lot @sarahyurick  for the reviews